### PR TITLE
fixed circular dependency

### PR DESCRIPTION
**Pull request by scottarver**
WARNING in Circular dependency detected:
../session-client/src/al-session.ts -> ../session-client/src/events/index.ts -> ../session-client/src/al-session.ts

WARNING in Circular dependency detected:
../session-client/src/events/index.ts -> ../session-client/src/al-session.ts -> ../session-client/src/events/index.ts

WARNING in Circular dependency detected:
../session-client/src/index.ts -> ../session-client/src/utilities/index.ts -> ../session-client/src/utilities/al-session-detector.ts -> ../session-client/src/index.ts

WARNING in Circular dependency detected:
../session-client/src/utilities/al-session-detector.ts -> ../session-client/src/index.ts -> ../session-client/src/utilities/index.ts -> ../session-client/src/utilities/al-session-detector.ts

WARNING in Circular dependency detected:
../session-client/src/utilities/index.ts -> ../session-client/src/utilities/al-session-detector.ts -> ../session-client/src/index.ts -> ../session-client/src/utilities/index.ts

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 export * from './types';
-export * from './events';
 export * from './utilities';
 export {
     AlSessionInstance,
     AlSession,
     AlSession as ALSession       /* deprecated */
 } from './al-session';
+export  *  from './al-session';
```
Comment (parky128) [inline src/index.ts]: two exports from `./al-session` now? won't the `export *` override the previous `export`?

Comment (scottarver) [reply src/index.ts]: true

Comment (parky128) [reply src/index.ts]: Ignore me, i see it doesnt!

Comment (parky128) [reply src/index.ts]: compiles down to the following in the typings:

```
export * from './types';
export * from './utilities';
export { AlSessionInstance, AlSession, AlSession as ALSession } from './al-session';
export * from './al-session';
```

So I think we are ok here

**Comment by parky128** [comment]
> This feels like a step back, organizationally. I would prefer to remove the session instance reference that the two offending events are passing; nothing that I am aware of uses them in any case.
> 
> Also, two space tabs...!!!

https://github.com/search?q=import+%7B+AlSessionInstance&type=Code

Yep nothing, the alertlogic results in that are an old public facing copy of ng-common-components thats not in use by anyone externally

**Comment by azverev** [comment]
### CI Service Job Number 1 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 1 -g`

[Logs for Job 1](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/1)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 2 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 2 -g`

[Logs for Job 2](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/2)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 3 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 3 -g`

[Logs for Job 3](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/3)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 4 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 4 -g`

[Logs for Job 4](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/4)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 5 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 5 -g`

[Logs for Job 5](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/5)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 6 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 6 -g`

[Logs for Job 6](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/6)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 7 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 7 -g`

[Logs for Job 7](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/7)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 8 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 8 -g`

[Logs for Job 8](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/8)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 9 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 9 -g`

[Logs for Job 9](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/9)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 10 Logs - FAILURE
`alps ci job-logs --organization alertlogic --repository session-client --job-num 10 -g`

[Logs for Job 10](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/10)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Comment by azverev** [comment]
### CI Service Job Number 11 Logs - SUCCESS
`alps ci job-logs --organization alertlogic --repository session-client --job-num 11 -g`

[Logs for Job 11](https://ui.pipelineservices.alertlogic.com/logs/alertlogic/session-client/11)

### Stages Run:
- PR Test


[Install and configure alps cli](https://alertlogic.atlassian.net/wiki/spaces/PSD/pages/776045105/Install+and+configure+alps+cli)

**Pull request merged**